flags/eigendaflags/cli.go

```diff
@@ -163,10 +181,96 @@ func CLIFlags(envPrefix, category string) []cli.Flag {
 			EnvVars:  []string{withEnvPrefix(envPrefix, "PUT_RETRIES")},
 			Category: category,
 		},
+	},
+		v2Flags(envPrefix, category)...,
+	)
+}
+
+func v2Flags(envPrefix, category string) []cli.Flag {
+	return []cli.Flag{
+		// EigenDA V2 specific flags //
+		&cli.BoolFlag{
+			Name:     V2Enabled,
+			Usage:    "Enable blob dispersal and retrieval against EigenDA v2 protocol",
+			EnvVars:  []string{withEnvPrefix(envPrefix, "V2_ENABLED")},
+			Required: false,
+		},
+		&cli.StringFlag{
+			Name:     CertVerifierAddrName,
+			Usage:    "Address of the EigenDABlobVerifier contract. Required for performing eth_calls to verify EigenDA certificates.",
+			EnvVars:  []string{withEnvPrefix(envPrefix, "CERT_VERIFIER_ADDR")},
+			Category: category,
+			Required: false,
+		},
+		&cli.DurationFlag{
+			Name:     RelayTimeoutName,
+			Usage:    "Timeout used when querying a relay for blob contents.",
```

Comment:
```suggestion
			Usage:    "Timeout used when querying a relay for blobs.",
```

Reply:
technically we should change this to payload

Reply:
Technically what's being retrieved from the relay _is_ the blob, and that's what the timeout applies to. Conversion from a blob to a payload happens after a successful blob retrieval.

We had discussed letting users be ignorant of the concept of a `blob`, so I'm a bit torn on technical correctness vs.  simple mental model for users here

Reply:
@litt3 trust me no one in the crypto space is ignorant to the concept of a blob.

Reply:
It's not a question of whether someone has heard of the generic concept of a `blob` before: it's about whether a user must be aware of the specific nuances of an *eigenDA blob*.